src/features/decorators.tsx

```diff
@@ -263,7 +264,7 @@ const settings = [
   "Context Enabled",
   "Hex Color Preview Enabled",
 ] as const;
-const DecoratorSettings = ({ isOpen, onClose }: RoamOverlayProps) => {
+const DecoratorSettings = ({ isOpen, onClose, extensionAPI }: RoamOverlayProps) => {
```

Comment:
We'll need to fix the type here: `RoamOverlayProps<{ extensionAPI: OnloadArgs["extensionAPI"]}>`

Reply:
I went a different route and remove the `addCommand` from `toggleFeature`
It made more sense to me to not "turn on and off" the feature every decoration save.
It also removes the need for type updates.  And this makes more sense to me as well, as `extensionAPI` shouldn't be required for `Decorations`
https://github.com/dvargas92495/roamjs-workbench/pull/391/commits/3eb2707ccff6a2f399f911e118dd048ae05daeab#diff-ea4e85188fe94af5ae70fed25db2d64551b1b568075d213927ed8f69a97fb202